bug report: slog-stdlog depends log::__private_api_log and slog-stdlog break with log v0.4.15

when updating log to 0.4.15, the `log::__private_api_log` change and slog-stdlog is broken

> ```
> error[E0061]: this function takes 4 arguments but 3 arguments were supplied
>     --> /home/sherlock/.cargo/registry/src/github.com-1ecc6299db9ec823/slog-stdlog-4.1.0/lib.rs:244:9
>      |
> 244  |         log::__private_api_log(format_args!("{}", lazy), level, &(target, info.module(), info.file(), info.line()));
>      |         ^^^^^^^^^^^^^^^^^^^^^^ ------------------------  -----  -------------------------------------------------- supplied 3 arguments
>      |         |
>      |         expected 4 arguments
>      |
> note: function defined here
>     --> /home/sherlock/.cargo/registry/src/github.com-1ecc6299db9ec823/log-0.4.15/src/lib.rs:1576:8
>      |
> 1576 | pub fn __private_api_log(
>      |        ^^^^^^^^^^^^^^^^^
> 
> 
> ```

although this `log::__private_api_log` is exported, we all know this is a private function and hope no one uses it

could slog team bump a new version to fix it?

--- a/lib.rs
+++ b/lib.rs
@@ -250,9 +250,32 @@ impl slog::Drain for StdLog {
         }
 
         let lazy = LazyLogString::new(info, logger_values);
-        // Please don't yell at me for this! :D
-        // https://github.com/rust-lang-nursery/log/issues/95
-        log::__private_api_log(format_args!("{}", lazy), level, &(target, info.module(), info.file(), info.line()));
+        /*
+         * TODO: Support `log` crate key_values here.
+         *
+         * This requires the log/kv_unstable feature here.
+         *
+         * Not supporting this feature is backwards compatible
+         * and it shouldn't break anything (because we've never had),
+         * but is undesirable from a feature-completeness point of view.
+         *
+         * However, this is most likely not as powerful as slog's own
+         * notion of key/value pairs, so I would humbly suggest using `slog`
+         * directly if this feature is important to you ;)
+         *
+         * This avoids using the private log::__private_api_log api function,
+         * which is just a thin wrapper around a `RecordBuilder`.
+         */
+        log::logger().log(
+            &log::Record::builder()
+                .args(format_args!("{}", lazy))
+                .level(level)
+                .target(target)
+                .module_path_static(Some(info.module()))
+                .file_static(Some(info.file()))
+                .line(Some(info.line()))
+                .build(),
+        );
 
         Ok(())
     }

